be/src/vec/functions/function_timestamp.cpp

```diff
@@ -1182,6 +1184,139 @@ class FunctionOtherTypesToDateType : public IFunction {
     }
 };
 
+struct FromIso8601DateV2 {
+    static constexpr auto name = "from_iso8601_date";
+
+    static bool is_variadic() { return true; } //todo : check is or not is_variadic
```

Comment:
does this function support variadic?

Reply:
don't support variadic. I forgot to delete this line.